build.gradle

```diff
@@ -51,9 +55,15 @@ dependencies {
     testCompile 'org.mockito:mockito-core:2.21.0'
     testCompile "com.squareup.okhttp3:mockwebserver:${okhttpclientVersion}"
     testCompile 'junit:junit:4.12'
+    testRuntime("org.junit.vintage:junit-vintage-engine:5.2.0")
```

Comment:
I think having both junit 4 and 5 on the classpath is not a good idea. I'd be in favor of upgrading everything to junit 5 but that's a bigger PR. Probably very straightforward to do.

Reply:
thanks, I created an issue for this https://github.com/stellar/java-stellar-sdk/issues/215